frontend/src/__tests__/cypress/cypress/e2e/acceleratorProfiles/manageAcceleratorProfiles.cy.ts

```diff
@@ -0,0 +1,151 @@
+    editTolerationModal.findCancelButton().click();
+    createAcceleratorProfile.getRow('Key', 'test-key').first();
+    createAcceleratorProfile.getRow('Key', 'test-key').first().findKebabAction('Edit').click();
+    editTolerationModal.findTolerationKeyInput().fill('test-update');
+    editTolerationModal.findTolerationSubmitButton().click();
+    createAcceleratorProfile.getRow('Key', 'test-update').first();
+
+    // test cancel clears fields
+    createAcceleratorProfile.getRow('Key', 'test-update').first().findKebabAction('Edit').click();
+    editTolerationModal.findCancelButton().click();
+    createAcceleratorProfile.findTolerationsButton().click();
+    createTolerationModal.findTolerationSubmitButton().should('be.disabled');
+    createTolerationModal.findCancelButton().click();
+
+    // test delete
+    createAcceleratorProfile.getRow('Key', 'test-update').first().findKebabAction('Delete').click();
+    createAcceleratorProfile.getRow('Key', 'test').findKebabAction('Delete').click();
+    createAcceleratorProfile.findTolerationsModalEmptyState().should('exist');
+  });
+
+  it('edit page has expected values', () => {
+    cy.intercept(
+      '/api/k8s/apis/dashboard.opendatahub.io/v1/namespaces/opendatahub/acceleratorprofiles/test-accelerator',
+      mockAcceleratorProfile({
+        name: 'test-accelerator',
+        displayName: 'Test Accelerator',
+        description: 'Test description',
+        identifier: 'nvidia.com/gpu',
+        tolerations: [
+          {
+            key: 'nvidia.com/gpu',
+            operator: TolerationOperator.EXISTS,
+            effect: TolerationEffect.NO_SCHEDULE,
+          },
+        ],
+      }),
+    );
+    editAcceleratorProfile.visit('test-accelerator');
+    editAcceleratorProfile.findAcceleratorNameInput().should('have.value', 'Test Accelerator');
+    editAcceleratorProfile.findIdentifierInput().should('have.value', 'nvidia.com/gpu');
+    editAcceleratorProfile.findDescriptionInput().should('have.value', 'Test description');
+    editAcceleratorProfile.getRow('Operator', 'Exists').first();
+    editAcceleratorProfile.getRow('Key', 'nvidia.com/gpu').first();
+    editAcceleratorProfile.getRow('Effect', 'NoSchedule').first();
+    editAcceleratorProfile.getRow('"Toleration Seconds"', '-').first();
+  });
+
+  it('invalid id in edit page', () => {
+    editAcceleratorProfile.visit('test-accelerator');
+    editAcceleratorProfile.findErrorText().should('exist');
+  });
+
+  it('one preset identifier is auto filled and disabled', () => {
+    identifierAcceleratorProfile.visit();
+    identifierAcceleratorProfile.findIdentifierInput().should('have.value', 'test-identifier');
+    identifierAcceleratorProfile.findIdentifierInput().should('be.disabled');
+  });
+
+  it('multiple preset identifiers show dropdown', () => {
+    identifierAcceleratorProfile.visit(true);
+    identifierAcceleratorProfile.findIdentifierField().findSelectOption('test-identifier').click();
```

Comment:
is this sama as the `identifierAcceleratorProfile.findIdentifierInput().should('have.value', 'test-identifier');` above? If yes, this can reuse the same function.

Reply:
Its a different selector